### ISSUE TITLE
The number of days away is incorrect for due dates

### ISSUE THREAD

**Issue by poikilos** [nextcloud/deck#2639]
version 1.2.2

The number of days away is incorrect (the number should not be rounded up).

**To Reproduce**
1. Early in the morning, set a due date late at night for another day
2. The number of days away is wrong (it rounds up, presumably since more than x days + 12 hours)

**Expected behavior**
Provide the number of days away without respect to hours (floor instead of round if using hours).
- Show hours separately if anything.

**Screenshots**
![kanban - says 5 days away, should say 4](https://user-images.githubusercontent.com/7557867/101616082-e7d4bf80-39dc-11eb-9c04-b4404546057f.png)
(says 5 days away, should say 4)

**Client details:**
 - OS: Windows 10
 - Browser Firefox
 - Version 83.0
 - Device: desktop

<details>
<summary>Server details</summary>
**Operating system**:
Ubuntu 20.04

**Web server:**
Nginx

**Database:**
PostgreSQL

**PHP version:**
7.3

**Nextcloud version:** (see Nextcloud admin page)
19.0.5

**Where did you install Nextcloud from:**
tarball (then upgraded via GUI several times)

**Signing status:**

```
No errors have been found.
```

**List of activated apps:**
N/A
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**
N/A
```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No
</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
N/A
```
Insert your Nextcloud log here
```

#### Browser log
```
N/A
```

</details>

**Comment by juliushaertl** [comment]
cc @stefan-niedermann I think that makes sense, not sure how you handle that in the android app but it think it would make sense to align here :)

**Comment by stefan-niedermann** [comment]
Yes, i agree. I will check how we solved it there and then post a proposal here tomorrow.
